### DISCUSSION TITLE
23.11.15 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#65] [Daily Scrum]
.

**Comment by Nine-JH** [comment]
* 오늘은 어떤 일을 했나요?
  * 정산 로직 수정
* 내일은 어떤 일을 하나요?
  * 음악에서 장르 정보 추출하기

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - 노래 저장 기능 구현
- 내일은 어떤 일을 하나요?
  - PR 넣기
  - 기존 기능과 연동

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 자현님의 도움을 받아.. 로그인 에러 해결!
- 내일은 어떤 일을 하나요?
    - PR
